### PR TITLE
Compatibility with fluentbit

### PR DESCRIPTION
**Pull request by bkupidura**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-      flow:
+    enabled: true
+    config:
+      label:
         monitoring:
+          name: MONITORING
           filter:
-            - pattern: 'docker.monitoring.{alertmanager,remote_storage_adapter,prometheus}.*'
+            parse_log:
+              pattern: 'docker.monitoring.{alertmanager,remote_storage_adapter,prometheus}.*'
```
Comment (epcim) [inline README.rst]: isnt this what is match? I would rather reword pattern to match (if I am right.

File: README.rst [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
-      enabled: true
-      flow:
+    enabled: true
+    config:
+      label:
         monitoring:
+          name: MONITORING
           filter:
-            - pattern: 'docker.monitoring.{alertmanager,remote_storage_adapter,prometheus}.*'
+            parse_log:
```
Comment (epcim) [inline README.rst:11]: compare with today fluentbit example
```
  fluentbit:
    config:
      filter:
        record_hostname.conf:                 <-- corespond to "parse_log", in fluentbit its the part for filename compose,
                                                                    unless "name:" is provided
          record_modifier:
            record: hostname ${linux:network:fqdn}
            match: '*'

```
may be I would suggest "parser" section. (as type is always required)

Comment (epcim) [reply README.rst:11]: thinking...

```
label:
  filter:
    parser:
       XYZ (without explicit type)
    parser2:
       type: parser
    record_tranformer:
       xyuz
```

**Review by dkalashnik — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (bkupidura) [reply README.rst]: In fluentd filters also use patterns (like match)

```
<filter docker.monitoring.{alertmanager,remote_storage_adapter,prometheus}.*>
  @type record_transformer
  remove_keys log
</filter>
<match docker.**>
  @type file
  path /tmp/flow-docker.log
</match>
```

File: README.rst (hunk 2 above)
Comment (bkupidura) [reply README.rst:11]: Regarding first one (parse_log and filename) - in label each section (input/filter/match) will not provide filename as whole label will land in single file, so filename is defined only for label. 
```
  config:
    label:
      monitoring: <--- filename
        input:
          input_name:
            params
           input_name2:
             params
        filter:
          filter_name:
            params
```

**Pull request closed without merging**